src/biogeochem/EDBGCDynMod.F90

```diff
@@ -180,6 +182,14 @@ subroutine EDBGCDyn(bounds,                                                    &
     if (decomp_method == century_decomp) then
        call decomp_rate_constants_bgc(bounds, num_soilc, filter_soilc, &
             soilstate_inst, temperature_inst, ch4_inst, soilbiogeochem_carbonflux_inst)
+    else if (decomp_method == mimics_decomp) then
+       call decomp_rates_mimics(bounds, num_soilc, filter_soilc, &
+            soilstate_inst, temperature_inst, cnveg_carbonflux_inst, ch4_inst, &
+            soilbiogeochem_carbonflux_inst, soilbiogeochem_carbonstate_inst)
+    else
```

Comment:
As above, remove the else with the warning. This should just be done at initialization.

Reply:
Removed `else`